demos/projects/ESPRESSIF/adu/config/demo_config.h

```diff
@@ -124,340 +124,83 @@
-    0x4F, 0x3A, 0xC7, 0x6E, 0x11, 0x3F, 0xE6, 0xE3, 0x16, 0x91,
-    0x84, 0xFB, 0x6D, 0x84, 0x7F, 0x34, 0x74, 0xAD, 0x89, 0xA7,
-    0xCE, 0xB9, 0xD7, 0xD7, 0x9F, 0x84, 0x64, 0x92, 0xBE, 0x95,
-    0xA1, 0xAD, 0x09, 0x53, 0x33, 0xDD, 0xEE, 0x0A, 0xEA, 0x4A,
-    0x51, 0x8E, 0x6F, 0x55, 0xAB, 0xBA, 0xB5, 0x94, 0x46, 0xAE,
-    0x8C, 0x7F, 0xD8, 0xA2, 0x50, 0x25, 0x65, 0x60, 0x80, 0x46,
-    0xDB, 0x33, 0x04, 0xAE, 0x6C, 0xB5, 0x98, 0x74, 0x54, 0x25,
-    0xDC, 0x93, 0xE4, 0xF8, 0xE3, 0x55, 0x15, 0x3D, 0xB8, 0x6D,
-    0xC3, 0x0A, 0xA4, 0x12, 0xC1, 0x69, 0x85, 0x6E, 0xDF, 0x64,
-    0xF1, 0x53, 0x99, 0xE1, 0x4A, 0x75, 0x20, 0x9D, 0x95, 0x0F,
-    0xE4, 0xD6, 0xDC, 0x03, 0xF1, 0x59, 0x18, 0xE8, 0x47, 0x89,
-    0xB2, 0x57, 0x5A, 0x94, 0xB6, 0xA9, 0xD8, 0x17, 0x2B, 0x17,
-    0x49, 0xE5, 0x76, 0xCB, 0xC1, 0x56, 0x99, 0x3A, 0x37, 0xB1,
-    0xFF, 0x69, 0x2C, 0x91, 0x91, 0x93, 0xE1, 0xDF, 0x4C, 0xA3,
-    0x37, 0x76, 0x4D, 0xA1, 0x9F, 0xF8, 0x6D, 0x1E, 0x1D, 0xD3,
-    0xFA, 0xEC, 0xFB, 0xF4, 0x45, 0x1D, 0x13, 0x6D, 0xCF, 0xF7,
-    0x59, 0xE5, 0x22, 0x27, 0x72, 0x2B, 0x86, 0xF3, 0x57, 0xBB,
-    0x30, 0xED, 0x24, 0x4D, 0xDC, 0x7D, 0x56, 0xBB, 0xA3, 0xB3,
-    0xF8, 0x34, 0x79, 0x89, 0xC1, 0xE0, 0xF2, 0x02, 0x61, 0xF7,
-    0xA6, 0xFC, 0x0F, 0xBB, 0x1C, 0x17, 0x0B, 0xAE, 0x41, 0xD9,
-    0x7C, 0xBD, 0x27, 0xA3, 0xFD, 0x2E, 0x3A, 0xD1, 0x93, 0x94,
-    0xB1, 0x73, 0x1D, 0x24, 0x8B, 0xAF, 0x5B, 0x20, 0x89, 0xAD,
-    0xB7, 0x67, 0x66, 0x79, 0xF5, 0x3A, 0xC6, 0xA6, 0x96, 0x33,
-    0xFE, 0x53, 0x92, 0xC8, 0x46, 0xB1, 0x11, 0x91, 0xC6, 0x99,
-    0x7F, 0x8F, 0xC9, 0xD6, 0x66, 0x31, 0x20, 0x41, 0x10, 0x87,
-    0x2D, 0x0C, 0xD6, 0xC1, 0xAF, 0x34, 0x98, 0xCA, 0x64, 0x83,
-    0xFB, 0x13, 0x57, 0xD1, 0xC1, 0xF0, 0x3C, 0x7A, 0x8C, 0xA5,
-    0xC1, 0xFD, 0x95, 0x21, 0xA0, 0x71, 0xC1, 0x93, 0x67, 0x71,
-    0x12, 0xEA, 0x8F, 0x88, 0x0A, 0x69, 0x19, 0x64, 0x99, 0x23,
-    0x56, 0xFB, 0xAC, 0x2A, 0x2E, 0x70, 0xBE, 0x66, 0xC4, 0x0C,
-    0x84, 0xEF, 0xE5, 0x8B, 0xF3, 0x93, 0x01, 0xF8, 0x6A, 0x90,
-    0x93, 0x67, 0x4B, 0xB2, 0x68, 0xA3, 0xB5, 0x62, 0x8F, 0xE9,
-    0x3F, 0x8C, 0x7A, 0x3B, 0x5E, 0x0F, 0xE7, 0x8C, 0xB8, 0xC6,
-    0x7C, 0xEF, 0x37, 0xFD, 0x74, 0xE2, 0xC8, 0x4F, 0x33, 0x72,
-    0xE1, 0x94, 0x39, 0x6D, 0xBD, 0x12, 0xAF, 0xBE, 0x0C, 0x4E,
-    0x70, 0x7C, 0x1B, 0x6F, 0x8D, 0xB3, 0x32, 0x93, 0x73, 0x44,
-    0x16, 0x6D, 0xE8, 0xF4, 0xF7, 0xE0, 0x95, 0x80, 0x8F, 0x96,
-    0x5D, 0x38, 0xA4, 0xF4, 0xAB, 0xDE, 0x0A, 0x30, 0x87, 0x93,
-    0xD8, 0x4D, 0x00, 0x71, 0x62, 0x45, 0x27, 0x4B, 0x3A, 0x42,
-    0x84, 0x5B, 0x7F, 0x65, 0xB7, 0x67, 0x34, 0x52, 0x2D, 0x9C,
-    0x16, 0x6B, 0xAA, 0xA8, 0xD8, 0x7B, 0xA3, 0x42, 0x4C, 0x71,
-    0xC7, 0x0C, 0xCA, 0x3E, 0x83, 0xE4, 0xA6, 0xEF, 0xB7, 0x01,
-    0x30, 0x5E, 0x51, 0xA3, 0x79, 0xF5, 0x70, 0x69, 0xA6, 0x41,
-    0x44, 0x0F, 0x86, 0xB0, 0x2C, 0x91, 0xC6, 0x3D, 0xEA, 0xAE,
-    0x0F, 0x84
-};
-#define democonfigROOT_CA_PEM root_cert_array
+#define democonfigROOT_CA_PEM \
```

Comment:
what call this democonfig prefix?

Reply:
The samples do. Example here:
https://github.com/Azure-Samples/iot-middleware-freertos-samples/blob/be6c0dfbbeb0fec07a8d53bc63bba65b71429f49/demos/sample_azure_iot/sample_azure_iot.c#L275-L276